python_modules/dagster/dagster/_core/definitions/definitions_class.py

```diff
@@ -44,7 +44,7 @@ class Definitions:
             }
         )
 
-    Create a set of definitions explicitly available and loadable by Dagster tools.
+    A set of definitions to be explicitly available and loadable by Dagster tools.
```

Comment:
[PEP](https://peps.python.org/pep-0257/#multi-line-docstrings) would recommend putting this at the top: 

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line. The summary line may be on the same line as the opening quotes or on the next line. The entire docstring is indented the same as the quotes at its first line (see example below).

Reply:
👍🏻